packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -0,0 +1,210 @@
+.rt-DataListRoot {
+  display: grid;
+  grid-template-columns: auto 1fr;
+}
```

Comment:
Nit: can we include this only when the data list is horizontal?

The layout debugging experience is a bit weird otherwise

<img width="821" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/bfa7100f-2737-4dea-9335-ee311f7af09f">

Reply:
Yep! Good call.